src/cohort-manager.js

```diff
@@ -0,0 +1,88 @@
+// const Cohort = require('../src/cohort.js')
+
+class CohortManager {
+  constructor() {
+    this.cohorts = []
+    this.student = []
+
+    // this.removedCohort = []
+  }
+
+  createCohorts(cohort) {
+    // Take cohort name & generate new cohort (from cohort class)
+    // this.name = name;
+    // Then save it in list of cohorts
+    // console.log()
+    this.cohorts.push(cohort)
+    // Return updated list of cohorts
+    return cohort.name
+  }
+
+  searchForCohort(cohortName) {
+    // const searchArray = []
+    // console.log('X: search: ', cohortName, this.cohorts)
+    for (let i = 0; i < this.cohorts.length; i++) {
+      if (cohortName === this.cohorts[i].name) {
+        // searchArray.push(this.cohorts[i])
+        return this.cohorts[i]
+      }
+    }
+    return false
+  }
+
+  removeCohort(cohortName) {
+    let removedCohort = []
+    // Go to cohort list & loop through to find the correct cohort(s)
+    for (let i = 0; i < this.cohorts.length; i++) {
```

Comment:
For loops are always the go-to, but maybe exploring other functions would be beneficial for you in the future:)
(e.g for...of loop, .filter, .find, .map... )